specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -5189,6 +5203,45 @@
       },
       "description": "Response from a List SynonymMaps request. If successful, it includes the full definitions of all synonym maps."
     },
+    "EncryptionKey": {
+      "properties": {
+        "keyVaultKeyName": {
+          "type": "string",
+          "description": "The name of the KeyVault key."
+        },
+        "keyVaultKeyVersion": {
+          "type": "string",
+          "description": "The version of the KeyVault key."
+        },
+        "keyVaultUri": {
+          "type": "string",
+          "description": "The URI of the KeyVault."
+        },
+        "accessCredentials": {
+          "$ref": "#/definitions/AzureActiveDirectoryApplicationCredentials",
+          "description": "The credentials to use to authenticate to the provided KeyVault. Can be left unspecified if MSI (Managed Service Identity) is to be used instead."
+        }
+      },
+      "required": [
+        "keyVaultKeyName",
+        "keyVaultKeyVersion",
+        "keyVaultUri"
+      ],
+      "description": "The configuration to use encryption with customer managed keys."
```

Comment:
perhaps this is a good place to add: "Specifying an encryption key to enable encryption with customer managed keys is only supported for services created on or after 01/01/2019" 
WDYT?

Reply:
As discussed offline, I intended on keeping all comments expressing "usage-related" requirements in the description of properties where we actually use them. So for example, I only express those requirements in the encryption key properties of synonym maps or indexes, since those are the scenarios that have those requirements.